setup.py

```diff
@@ -60,8 +60,10 @@
     test_suite='tests',
     zip_safe=zip_safe,
     install_requires=[
+        "email_validator>=1.0.2",
         "python-dateutil>=2.4",
         "six",
+        "ukpostcodeparser>=1.1.1",
```

Comment:
With this dependency on `ukpostcodeparser`, our internal tools started throwing red flags because it's not properly licensed.

https://github.com/hamstah/ukpostcodeparser/issues/7

It's also not clear why this is in `requires` and not `test_requires`.

Reply:
you're right, it should go in `test_requires`, since it's used only for tests